bin/omarchy-theme-menu

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+# Theme menu for Omarchy
+# Provides an interactive dmenu to switch between available themes
+
+THEMES_DIR="$HOME/.local/share/omarchy/themes/"
+CURRENT_THEME_LINK="$HOME/.config/omarchy/current/theme"
+
+# Show theme selection menu and apply changes
+show_theme_menu() {
+  # Get current theme name
+  if [[ -L "$CURRENT_THEME_LINK" ]]; then
+    CURRENT_THEME_NAME=$(basename "$(realpath "$CURRENT_THEME_LINK")")
+  else
+    CURRENT_THEME_NAME="unknown"
+  fi
```

Comment:
Think you should always be able to use basename/realpath here, even if the theme is an actual directory, not a symlink.